execution_nodes/test/functional/execute_nodeTest.js

```diff
@@ -0,0 +1,60 @@
+const chai = require('chai');
+const dirtyChai = require('dirty-chai');
+const chaiHttp = require('chai-http');
+const nock = require('nock');
+const executeNode = require('../../execute_node.js');
+const { submissions } = require('./data/submission.json');
+const { results } = require('./data/result.json');
+const { check } = require('../../../util/environmentCheck.js');
+
+check('ENCONFIG');
+check('ENSCORES');
+// eslint-disable-next-line import/no-dynamic-require
+const conf = require(`../../${process.env.ENCONFIG}`);
+// eslint-disable-next-line import/no-dynamic-require
+const scores = require(`../../${process.env.ENSCORES}`);
+const enUrl = `https://${conf.execution_node.hostname}:${conf.execution_node.port}`;
+const lbUrl = `https://${conf.load_balancer.hostname}:${conf.load_balancer.port}`;
+chai.should();
+chai.use(dirtyChai);
+chai.use(chaiHttp);
+
+describe('HTTP call for connection check is successful', () => {
+  it('Connection check returns true', (done) => {
+    chai.request(enUrl)
+      .get('/connectionCheck')
+      .end((err, res) => {
+        (err === null).should.be.true();
+        res.body.should.equal(true);
+        done();
+      });
+  });
+});
+
+describe('HTTP call for request run is successful', () => {
+  it('Request run is successful for a normal evaluation', (done) => {
+    const lb = nock(lbUrl)
+      .post('/sendScores')
+      .reply(200, (uri, requestBody) => {
+        requestBody.should.have.all.keys('node_details', 'submission_details');
+        requestBody.node_details.should.deep.equal(scores.node_details);
+        requestBody.submission_details.should.deep.equal(results[0]);
+        done();
+        return true;
+      });
+
+    chai.request(enUrl)
+      .post('/requestRun')
+      .set('content-type', 'application/json')
+      .send(submissions[0])
+      .end((err, res) => {
+        (err === null).should.be.true();
+        res.body.should.equal(true);
+      });
+  });
```

Comment:
Other tests that can be added are:
* execution errors resulting in missing results/ directory
* language executables like java / python missing resulting in execution errors
* config files missing

As far as possible, we should reuse the existing test data instead of adding the new one.